graspologic/embed/base.py

```diff
@@ -95,10 +100,11 @@ def _reduce_dim(self, A):
             n_iter=self.n_iter,
         )
 
+        square = A.shape[0] == A.shape[1]
         self.n_components_ = D.size
         self.singular_values_ = D
         self.latent_left_ = U @ np.diag(np.sqrt(D))
-        if not is_almost_symmetric(A):
+        if square and not is_almost_symmetric(A):
```

Comment:
Maybe the square variable is redundant?

Reply:
I added that in because I think I found some edge case where it actually wasn't redundant, but I'm forgetting what that edge case was now. I'll delete that if I don't remember what it was in the next day or two, lol

Reply:
ah ok. I asssume then you found the edge case this resolved?